test/test.ts

```diff
@@ -0,0 +1,13 @@
+/*
+	File: test/test.ts
+	cpuabuse.com
+*/
+
+/**
+ * Performs tests.
+ */
+
+import { unitTest } from "./unit/test";

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "./unit/test"](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5042286)